DeBooxUpx.py

```diff
@@ -87,3 +88,18 @@ def deUpx(self, inputFileName: str, outputFileName: str):
         self.deUpxSteam(inputFile, outputFile)
         inputFile.close()
         outputFile.close()
+
+
+if __name__ == '__main__':
+    import sys
+    if len(sys.argv) >= 2:
+        device_name = sys.argv[1].capitalize()
```

Comment:
`capitalize()` may convert some device names to incorrect forms, for example `NovaPro` to `Novapro`

Reply:
Done, I decided to print supported devices when incorrect key is passed, is that alright?

Reply:
Works well!